docs/source/asr/configs.rst

```diff
@@ -546,6 +547,38 @@ The encoder section includes the details about the RNN-based encoder architectur
 config files and also :ref:`nemo.collections.asr.modules.RNNEncoder <rnn-encoder-api>`.
 
 
+CTC Configurations
```

Comment:
@titu1994 not sure if that's the best place to put these docs

Reply:
It is for the configuration of ASR models, doesn't look like that bad to me.

Reply:
Same, looks fine